visualizations/frontend/other/TResetFilters.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <TTextButton @click="handleClick">Reset Filters</TTextButton>
+</template>
+
+<script>
+export default {
+  data() {
+    is_filter: true
+  },
+  methods: {
+    handleClick(e) {
+      console.log('handle the cliiiiick!', e)
+      this.unsetAllFilters()
```

Comment:
If I put this component in 'library' it complains that 'unsetAllFilters' is not a function. My assumption is that the VisualizationMixin is not available and can't be called when it is not in the visualizations folder. 

When the component is in the 'visualizations' folder, it has the HeightWrapper applied, which makes the component very big. `is_filter` seems to be the current hack for avoiding the HeightWrapper. When I use `is_filter`, however, the code throws two errors:

```
TypeError: e is undefined // for the 'e' argument to handleClick
ReferenceError: is_filter is not defined
```

I could do with some help understanding how where this component should sit, or if we need to refactor the code in topcoat-core.